lib/Controller/AssistantController.php

```diff
@@ -33,9 +34,10 @@ public function __construct(
 	 */
 	#[NoAdminRequired]
 	#[NoCSRFRequired]
-	#[BruteForceProtection(action: 'taskResultPage')]
+	#[BruteForceProtection(action: 'taskResults')]
```

Comment:
I guess I was thinking this from the point of view of an attacker. With two different actions I could get twice the number of attempts with different actions. It may well be that I've misunderstood the functioning of the brute force protection as well...

Reply:
... But do we need brute force protection here anymore since the service method only returns the task if it's owned by the user?

Reply:
Yeah i thought the same, we don't need it if it's not a public endpoint. Let's remove it.

Reply:
Ok, I also blocked the scheduling/running of tasks with "userId === null" while at it.